test/onnx/onnx_import.in.cpp

```diff
@@ -0,0 +1,1412 @@
+              {5.25626399e-07f, 1.42880069e-06f, 3.88388295e-06f, 1.05574884e-05f, 2.86982290e-05f},
+              {7.80098743e-05f, 2.12052824e-04f, 5.76419338e-04f, 1.56687021e-03f, 4.25919482e-03f},
+              {1.15776919e-02f,
+               3.14714295e-02f,
+               8.55482149e-02f,
+               2.32544158e-01f,
+               6.32120559e-01f}}})
+            .get_vector();
+
+    auto result_vectors = execute(function, inputs, "${BACKEND_NAME}");
+    EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_sub)
+{
+    auto function =
+        onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/sub.prototxt"));
+
+    Inputs inputs;
+    inputs.emplace_back(test::NDArray<float, 3>({{{1, 2, 3}}}).get_vector());
+
+    inputs.emplace_back(test::NDArray<float, 3>({{{4, 5, 7}}}).get_vector());
+
+    auto expected_output = test::NDArray<float, 3>({{{-3, -3, -4}}}).get_vector();
+
+    auto result_vectors = execute(function, inputs, "${BACKEND_NAME}");
+    EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_div)
+{
+    auto function =
+        onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/div.prototxt"));
+
+    Inputs inputs;
+    inputs.emplace_back(test::NDArray<float, 3>({{{1, 2, 3}}}).get_vector());
+
+    inputs.emplace_back(test::NDArray<float, 3>({{{1, 4, 12}}}).get_vector());
+
+    auto expected_output = test::NDArray<float, 3>({{{1, 0.5, 0.25}}}).get_vector();
+
+    auto result_vectors = execute(function, inputs, "${BACKEND_NAME}");
+    EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_add_bcast)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/add_bcast.prototxt"));
+
+    Inputs inputs;
+    inputs.emplace_back(test::NDArray<float, 3>(
+                            {{{1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}},
+                             {{1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}},
+                             {{1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}}})
+                            .get_vector());
+
+    inputs.emplace_back(test::NDArray<float, 1>({1, 2, 3, 4, 5}).get_vector());
+
+    Outputs expected_output{
+        test::NDArray<float, 4>(
+            {{{{2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}},
+              {{2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}},
+              {{2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}}}})
+            .get_vector()};
+
+    Outputs outputs{execute(function, inputs, "${BACKEND_NAME}")};
+    EXPECT_TRUE(test::all_close_f(expected_output.front(), outputs.front()));
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_reduce_log_sum)
```

Comment:
Maybe reduce tests should be in another file?

Reply:
Agreed, but I think we can do this in the future.